src/Report/EnabledMembers.php

```diff
@@ -31,7 +31,7 @@ class EnabledMembers extends Report
      * Cached registered methods fetched for the current list of members. This should only be populated at render time
      * as methods will be fetched for the current "records" on the report
      *
-     * @var DataList|null
+     * @var DataList<RegisteredMethod>|ArrayList<RegisteredMethod>|null
```

Comment:
```suggestion
     * @var SS_List<RegisteredMethod>|null
```

Reply:
Looks like it should actually be `ArrayList<RegisteredMethod>|null` so I'll change to that. I made a mistake when sight-reading `getRegisteredMethodsForRecords()` (which is the only place where it's set) before.

Reply:
Done